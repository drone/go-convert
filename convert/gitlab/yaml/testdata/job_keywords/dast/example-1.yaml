# https://docs.gitlab.com/ee/ci/yaml/#dast_configuration

stages:
  - build
  - dast

include:
  - template: DAST.gitlab-ci.yml

dast:
  dast_configuration:
    site_profile: "Example Co"
    scanner_profile: "Quick Passive Test"