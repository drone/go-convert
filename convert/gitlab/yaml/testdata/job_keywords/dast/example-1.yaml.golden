dast:
    dast_configuration:
        site_profile: Example Co
        scanner_profile: Quick Passive Test
include:
    - template: DAST.gitlab-ci.yml
stages:
    - build
    - dast
